### PR TITLE
[zh-cn] sync /docs/concepts/security/cloud-native-security.md

### PR DESCRIPTION
**Pull request by RifeWang**
### Issue

Closes: #

### CODE REVIEW

File: content/zh-cn/docs/concepts/security/cloud-native-security.md [hunk 1]
```diff
@@ -180,11 +180,11 @@ layers expect.
...
 <!--
-The Runtime phase comprises three critical areas: [compute](#protection-runtime-compute),
-[access](#protection-runtime-access), and [storage](#protection-runtime-storage).
+The Runtime phase comprises three critical areas: [access](#protection-runtime-access),
+[compute](#protection-runtime-compute), and [storage](#protection-runtime-storage).
 -->
-运行阶段包含三个关键领域：[计算](#protection-runtime-compute)，
-[访问](#protection-runtime-access)和[存储](#protection-runtime-storage)。
+运行阶段包含三个关键领域：[访问](#protection-runtime-access)，
+[计算](#protection-runtime-compute)和[存储](#protection-runtime-storage)。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/security/cloud-native-security.md:187]:
```suggestion
运行阶段包含三个关键领域：[访问](#protection-runtime-access)、
[计算](#protection-runtime-compute)和[存储](#protection-runtime-storage)。
```

**Review by windsonsea — COMMENTED** [review]
nit

File: content/zh-cn/docs/concepts/security/cloud-native-security.md (hunk 1 above)
Comment (RifeWang) [reply content/zh-cn/docs/concepts/security/cloud-native-security.md]: fixed

**Pull request merged**